tests/routes_tests/test_audit_status.py

```diff
@@ -77,6 +71,12 @@ def test_audit_status(client, election_id):
                             "name": "Audit Board #1",
                             "passphrase": assert_is_passphrase,
                         },
+                        {
+                            "id": audit_board_id_2,
+                            "members": [],
+                            "name": "audit board #2",
+                            "passphrase": assert_is_passphrase,
+                        },
```

Comment:
Order changed by adding the db unique constraint. I confirmed this by adding the db constraint on master and running the tests.

Reply:
should there be a defined order here that doesn't depend on the index? Meaning, are we not issuing an `order by`?

Reply:
You wrote these endpoints hehe. I can go try to add an ordering if you'd like

Reply:
Added: 540f5f2dcb003ed68a068173fd16325f483a31ed